source/install/prepare-mattermost-database.rst

```diff
@@ -204,3 +207,43 @@ When the PostgreSQL database is installed, and the initial setup is complete, yo
 .. important::
 
   If you are upgrading a major version of Postgres, it is essential that ``ANALYZE VERBOSE`` is run on the database post upgrade. This is necessary to re-populate the ``pg_statistics`` table used to generate optimal query plans. The database performance might suffer if this step is not done.
+
+
+Minimum supported version policy
+---------------------------------
+
+To make planning easier and ensure your Mattermost deployment remains fast and secure, we are introducing a policy for updating the minimum supported version of PostgreSQL. The oldest supported PostgreSQL version Mattermost supports will match the oldest version supported by the PostgreSQL community. This ensures you benefit from the latest features and security updates.
+
+This policy change takes effect from Mattermost v10.6, where the minimum PostgreSQL version required will be PostgreSQL 13. This aligns with the PostgreSQL community's support policy, which provides 5 years of support for each major version.
+
+.. note::
+
+  Mattermost v10.6 is not an :ref:`Extended Support Release (ESR) <about/release-policy:extended support releases>`. Going forward, this database version support policy will only apply to ESR releases.
+
+When a PostgreSQL version reaches its end of life (EOL), Mattermost will require a newer version starting with the next scheduled ESR release. This means the following future PostgreSQL minimum version increases as follows:
+
++-----------------------------------------------------------+------------------+--------------------------------+
+| **Mattermost Version**                                    | **Release Date** | **Minimum PostgreSQL Version** |
++===========================================================+==================+================================+
+| :ref:`v9.11 ESR <release-v9-11-extended-support-release>` | 2024-8-15        | 11.x                           |
++-----------------------------------------------------------+------------------+--------------------------------+
+| v10.5 ESR                                                 | 2025-2-15        | 11.x                           |
++-----------------------------------------------------------+------------------+--------------------------------+
+| v10.6                                                     | 2025-3-15        | 13.x                           |
++-----------------------------------------------------------+------------------+--------------------------------+
+| v10.11 ESR                                                | 2025-7-15        | 13.x                           |
```

Comment:
I think we are forecasting an ESR in August 2025 (6 months after the Feb 2025 ESR).

So the date for v10.11 should be 2025-08-15

Reply:
@amyblais  can you confirm that forecasting?  Things can change, but if we follow the pattern from the last year this would be the timeline.

Reply:
v10.11 ESR on 2025-08-15 is correct.